src/backend/distributed/utils/node_metadata.c

```diff
@@ -381,8 +381,8 @@ RemoveNodeFromCluster(char *nodeName, int32 nodePort, bool forceRemove)
 
 	nodeDeleteCommand = NodeDeleteCommand(workerNode->nodeId);
 
-	/* make sure we don't have any open connections */
-	CloseNodeConnections(nodeName, nodePort);
+	/* make sure we don't have any open session lifespan connections */
```

Comment:
s/open session lifespan connects/lingering session lifespan connections/? Since they're currently open and stay open...